bootstrap/039_reference_management.sql

```diff
@@ -17,7 +17,8 @@ begin
        select system$set_reference(:ref_name, :ref_or_alias);
         if (ref_name = 'OPSCENTER_API_INTEGRATION') then
             insert into internal.reference_management (ref_name, operation, ref_or_alias) values (:ref_name, 'Running external funcitons setup proc.', :ref_or_alias);
-            call admin.setup_external_functions();
+            call admin.setup_register_tenant_func();
```

Comment:
Other approach is I tried to wait long enough (> 5 min) to see if it gets fixed after in this callback. But external function definition never changed to the one in admin.setup_register_tenant_func